src/MomentumPyClient/ui.py

```diff
@@ -34,14 +30,43 @@ def get_container_definitions(self):
             "brown",
             "cyan",
             "gray",
+            "Cyan",
+            "Magenta",
+            "Teal",
+            "Pink",
+            "Lime",
+            "Lavender",
+            "Beige",
+            "Maroon",
+            "Mint",
+            "Peach",
+            "Navy",
+            "Olive",
+            "Coral",
         ]
-        i = 0
-        for container in self.containers:
-            container["color"] = color_names[i]
-            i += 1
-            if i >= len(color_names):
-                i = 0
-        return self.containers
+        self.color_dict = {}
+        self.color_index = 0
+        containers = self.ws.get_container_definitions()
+        # Create a dictioary to lookup the color
+        for container in containers:
+            #            self.get_container_color(container["Name"])
+            self.get_container_color(container["InventoryTemplateName"])
+
+    def set_container_colors(self, color_dict: dict):
+        self.color_dict = color_dict
+
+    def get_container_color(self, container_name):
+        if container_name is None:
```

Comment:
Wouldn't it be better to initialize self.color_dict with {None: "blue"} in line 47? That way, it can be overwritten, should we want to.